### PR TITLE
Fix statistics FAQ link (EXPOSUREAPP-4900)

### PR DESCRIPTION
**Pull request by ralfgehrer**

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-tr/strings.xml [hunk 1]
```diff
@@ -1330,8 +1330,8 @@
     <string name="statistics_explanation_seven_day_r_value_title">"7 Günlük R Değeri"</string>
     <!-- XTXT: Explanation screen seven day r-value text -->
     <string name="statistics_explanation_seven_day_r_value_text">"Reprodüksiyon değeri R, enfekte olan bir kişinin ortalama olarak kaç kişiyi enfekte ettiğini belirtir. Geçerli değerde, en fazla son 5 günün verileri dikkate alınır.\n\nDaha fazla bilgi için SSS bölümüne bakın:\nistatistikler için SSS."</string>
-    <!-- XTXT: Explanation screen seven day r-value link label -->
-    <string name="statistics_explanation_seven_day_r_link_label">"SSS bölümüne bakın."</string>
+    <!-- XTXT: Explanation screen seven day r-value link label  - the characters have to match the last part of the text label above -->
+    <string name="statistics_explanation_seven_day_r_link_label">"istatistikler için SSS."</string>
```
Comment (ralfgehrer) [inline Corona-Warn-App/src/main/res/values-tr/strings.xml:1334]: Needs to be updated in the translation system. I let Albert know.

File: Corona-Warn-App/src/main/res/values-bg/strings.xml [hunk 2]
```diff
@@ -1330,7 +1330,7 @@
     <string name="statistics_explanation_seven_day_r_value_title">"7-дневно репродуктивно число R"</string>
     <!-- XTXT: Explanation screen seven day r-value text -->
     <string name="statistics_explanation_seven_day_r_value_text">"Репродуктивното число R указва средно колко души заразява едно заразено лице. Текущата стойност взема предвид данните от последните 5 дни. \n\nПовече информация може да намерите в раздел ЧЗВ.\nЧЗВ относно статистиката."</string>
-    <!-- XTXT: Explanation screen seven day r-value link label -->
+    <!-- XTXT: Explanation screen seven day r-value link label - the characters have to match the last part of the text label above -->
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values-bg/strings.xml:1333]: Only add this to german, will be "delivered to all other languages.

File: Corona-Warn-App/src/main/res/values-pl/strings.xml [hunk 3]
```diff
@@ -1330,7 +1330,7 @@
     <string name="statistics_explanation_seven_day_r_value_title">"7-dniowa wartość R"</string>
     <!-- XTXT: Explanation screen seven day r-value text -->
     <string name="statistics_explanation_seven_day_r_value_text">"Współczynnik reprodukcji R wskazuje liczbę osób, którą zaraża przeciętna osoba zakażona. Bieżąca wartość uwzględnia dane z ostatnich 5 dni.\n\nWięcej informacji znajdziesz w odpowiedziach na często zadawane pytania:\nCzęsto zadawane pytania dotyczące statystyk."</string>
-    <!-- XTXT: Explanation screen seven day r-value link label -->
+    <!-- XTXT: Explanation screen seven day r-value link label  - the characters have to match the last part of the text label above -->
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values-pl/strings.xml:1333]: same as above

File: Corona-Warn-App/src/main/res/values-ro/strings.xml [hunk 4]
```diff
@@ -1330,7 +1330,7 @@
     <string name="statistics_explanation_seven_day_r_value_title">"Valoarea R pe 7 zile"</string>
     <!-- XTXT: Explanation screen seven day r-value text -->
     <string name="statistics_explanation_seven_day_r_value_text">"Numărul de reproducție R indică câte persoane a infectat mai departe, în medie, o persoană infectată. Valoarea curentă ia în considerare cel mult ultimele 5 zile.\n\nPentru alte informații, consultați întrebările frecvente:\nÎntrebări frecvente despre statistică."</string>
-    <!-- XTXT: Explanation screen seven day r-value link label -->
+    <!-- XTXT: Explanation screen seven day r-value link label  - the characters have to match the last part of the text label above -->
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values-ro/strings.xml:1333]: same as above

File: Corona-Warn-App/src/main/res/values-tr/strings.xml [hunk 5]
```diff
@@ -1330,8 +1330,8 @@
     <string name="statistics_explanation_seven_day_r_value_title">"7 Günlük R Değeri"</string>
     <!-- XTXT: Explanation screen seven day r-value text -->
     <string name="statistics_explanation_seven_day_r_value_text">"Reprodüksiyon değeri R, enfekte olan bir kişinin ortalama olarak kaç kişiyi enfekte ettiğini belirtir. Geçerli değerde, en fazla son 5 günün verileri dikkate alınır.\n\nDaha fazla bilgi için SSS bölümüne bakın:\nistatistikler için SSS."</string>
-    <!-- XTXT: Explanation screen seven day r-value link label -->
-    <string name="statistics_explanation_seven_day_r_link_label">"SSS bölümüne bakın."</string>
+    <!-- XTXT: Explanation screen seven day r-value link label  - the characters have to match the last part of the text label above -->
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values-tr/strings.xml:1333]: same as above

File: Corona-Warn-App/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -1345,7 +1345,7 @@
     <string name="statistics_explanation_seven_day_r_value_title">"7-Day R Value"</string>
     <!-- XTXT: Explanation screen seven day r-value text -->
     <string name="statistics_explanation_seven_day_r_value_text">"The reproduction figure, R, indicates how many people an average infected person has infected further. The current value takes data up to the last 5 days into account.\n\nFor more information, refer to the FAQ:\nFAQ for the statistics."</string>
-    <!-- XTXT: Explanation screen seven day r-value link label -->
+    <!-- XTXT: Explanation screen seven day r-value link label  - the characters have to match the last part of the text label above -->
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values/strings.xml:1348]: same as above

File: Corona-Warn-App/src/main/res/values-en/strings.xml [hunk 7]
```diff
@@ -1330,7 +1330,7 @@
     <string name="statistics_explanation_seven_day_r_value_title">"7-Day R Value"</string>
     <!-- XTXT: Explanation screen seven day r-value text -->
     <string name="statistics_explanation_seven_day_r_value_text">"The reproduction figure, R, indicates how many people an average infected person has infected further. The current value takes data up to the last 5 days into account.\n\nFor more information, refer to the FAQ:\nFAQ for the statistics."</string>
-    <!-- XTXT: Explanation screen seven day r-value link label -->
+    <!-- XTXT: Explanation screen seven day r-value link label  - the characters have to match the last part of the text label above -->
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values-en/strings.xml:1333]: same as above

**Review by harambasicluka — CHANGES_REQUESTED** [review]
I wouldn't add the comments for all languages except german, they will be part of the next delivery.

**Review by harambasicluka — APPROVED** [review]

**Review by mtwalli — APPROVED** [review]
LGTM , tested on Samsung S8 Android 9

**Review by SabineLoss — APPROVED** [review]
UA reviewed

**Pull request merged**